mill

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env sh
```

Comment:
This script is coming from the Mill releases.
It is a utility to allow people to run Mill without having to install it.